sycl/unittests/scheduler/InOrderQueueSyncCheck.cpp

```diff
@@ -72,35 +72,30 @@ TEST_F(SchedulerTest, InOrderQueueSyncCheck) {
   {
     LimitedHandlerSimulation MockCGH;
     EXPECT_CALL(MockCGH, depends_on).Times(0);
-    Queue->finalizeHandler<LimitedHandlerSimulation>(
-        MockCGH, detail::CG::CGTYPE::CodeplayHostTask, Event);
+    Queue->finalizeHandler<LimitedHandlerSimulation>(MockCGH, Event);

```

Comment:
Actually, most of these checks are identical now, so the test needs to be trimmed down. I'd rather do so in a follow up PR though to fix post-commit ASAP.